src/rmutil/args.c

```diff
@@ -159,4 +172,57 @@ int AC_GetVarArgs(ArgsCursor *ac, ArgsCursor *dst) {
   dst->offset = 0;
   AC_AdvanceBy(ac, nargs);
   return 0;
+}
+
+const ACArgSpec *AC_ParseArgSpec(ArgsCursor *ac, ACArgSpec *spec, int *err) {
+  const char *s = NULL;
+  size_t n;
+  if (AC_GetString(ac, &s, &n, AC_F_NOADVANCE) != AC_OK) {
+    // WAT!?
```

Comment:
If you think that can not happened then put an assert here ..

Reply:
Notice also that this is an example where you return NULL (when it actually kind of error), the function which calls it will try to parse this arg itself ..